test/core/bad_client/bad_client.cc

```diff
@@ -66,10 +66,10 @@ static void set_done_write(void* arg, grpc_error_handle /*error*/) {
 static void server_setup_transport(void* ts, grpc_transport* transport) {
   thd_args* a = static_cast<thd_args*>(ts);
   grpc_core::ExecCtx exec_ctx;
-  a->server->core_server->SetupTransport(transport,
-                                         /*accepting_pollset=*/nullptr,
-                                         a->server->core_server->channel_args(),
-                                         /*socket_node=*/nullptr);
+  (void)a->server->core_server->SetupTransport(
```

Comment:
I think that most of the ocurrences in the test tree should probably assert that the result is `GRPC_ERROR_NONE` instead of just ignoring the result.

For non-gtest files, you can use `GPR_ASSERT(GPR_LOG_IF_ERROR(...))`.

For gtest files, you can use either `ASSERT_EQ()` or `EXPECT_EQ()`, depending on the context.  And you can add `<< grpc_error_std_string(error)` at the end so that if the call does return an error, it will print a useful message.